packages/data-models/deployment.model.ts

```diff
@@ -140,6 +141,7 @@ export const DEPLOYMENT_CONFIG_EXAMPLE_DEFAULTS: IDeploymentConfig = {
     custom_ts_files: [],
     task_cache_path: "./tasks",
   },
+  favicon_asset: "",
```

Comment:
nit(non-blocking) might be worth putting the default path here just to save having to use conditionals in the seo service

Reply:
I think there would still need to be a conditional/check somewhere because the asset path gets a different prefix depending on whether the default asset is being used, as the custom asset sits inside the `app_data/assets/` dir where the default asset doesn't: `"assets/app_data/assets/<custom_favicon_path>"` vs `"assets/icon/favicon.png"`.